docs/source/streamflow-public-testnet.md

```diff
@@ -315,6 +315,82 @@ If you observe the `ticket faceValue higher than max faceValue` error in the bro
+
+```
+Sending build context to Docker daemon  5.904MB
+Step 1/4 : FROM verifier:v1
+ ---> fe5d1924dab5
+Step 2/4 : COPY /api ./scripts
+ ---> Using cache
+ ---> bc21bfb203ef
+Step 3/4 : ENTRYPOINT ["python"]
+ ---> Using cache
+ ---> 816b804bc7e7
+Step 4/4 : CMD ["scripts/api.py"]
+ ---> Using cache
+ ---> 548f58689798
+Successfully built 548f58689798
+Successfully tagged verifier-api:v1
+Verifier server listening in port 5000
+```
+
+If you are running the verifier on the same machine as the broadcaster, then you can run the below command in order to connect the broadcaster with the verifier:
+
+```
+livepeer -network rinkeby -broadcaster -verifierAddr http://localhost:5000/verify -verifierPath ~/verification-classifier/stream
+```
+
+The broadcaster logs should indicate the address of the verifier being used:
+
+```
+Using the Epic Labs classifier for verification at http://localhost:5000/verify
+```
+
+The value of `-verifierAddr` should be the address (`http://<IP>:<port>/verify`) that the verifier is receiving requests at. In this case, since the verifier and broadcaster are on the same machine all requests can be received on `localhost`.
+
+The value of `-verifierPath` should be the path to the volume that the verifier will read segment data from. By default, the verifier will read segment data from a volume mounted on the `/stream` directory within the `verification-classifier` project. So, if you started the verifier from within `~/verification-classifier` the value for `-verifierPath` should be `~/verification-classifier/stream`.
+
+If you are running the verifier on a separate machine from the broadcaster, you will need to make sure that the verifier endpoint is accessible by the broadcaster and that the verifier's volume is accessible over the network.
+
+One way to make the verifier endpoint accessible to the broadcaster without making it publicly accessible is to create an SSH tunnel from the broadcaster machine to the verifier machine.
+
+You can create an SSH tunnel with the following command run from the broadcaster machine:
+
+```
+ssh -N -L 5000:127.0.0.1:5000 -i <SSH_KEY_FILE> <USER>@<HOSTNAME>
+```
+
+Note that the command will not output anything if it is successfuly run.
+
+One way to make the verifier's volume accessible over the network is to mount the verifier's volume on the broadcaster machine over an SSH connection using [sshfs](https://github.com/libfuse/sshfs).
+
+You can mount the verifier's volume on the broadcaster machine by running the following command on the broadcaster machine:
+
+```
+sshfs -o IdentityFile=<SSH_KEY_FILE> <USER>@<HOSTNAME>:<FOLDER_ON_VERIFIER> <FOLDER_ON_BROADCASTER>
+```
+
+After the SSH tunnel is setup and the remote volume is mounted, you can run the following command to start the broadcaster:
```

Comment:
I'd go more imperatively here and omit "You can"

Reply:
Fixed in 14b0c3d